presto-native-execution/README.md

```diff
@@ -54,9 +50,10 @@ Makefile targets exist for showing, fixing and checking formatting, license
 headers and clang-tidy warnings.  These targets are shortcuts for calling
 `./scripts/check.py`.  
 
-CircleCi runs `make format-check`, `make header-check` and `make tidy-check` as
-part of our continious integration.  Pull requests should pass format-check and
```

Comment:
Do we actually run 'make tidy-check'? It wasn't part of CircleCI config, but maybe it was added to GitHub actions. BTW, where can I see GitHub Actions config?

Reply:
@mbasmanova, I don't know, we can check this in this PR. :) ... I don't see anything like that.
This is from what was in readme before.
I can talk to @mshang816 about this and we can update this later.

Reply:
I think we don't run tidy-check. Let's remove it from the README.

Reply:
@mbasmanova Maybe we don't even need all these comments about formatting and stuff, just mention the clang format and that just all checks need to be green before approving and merging.
I just tried to keep 'status quo' with all the old info in the readme that is not explicitly wrong.

Reply:
I agree. It is useful for folks to know that they can use 'make format-fix' and 'make header-fix' commands to fix formatting issues though. In Velox. this is documented in CODING_STYLE.md: https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#code-formatting-headers-and-licenses

Perhaps, we can add that to this README as well.

Reply:
BTW, do we have CODING_STYLE.md for presto_cpp? Or do we have a section on C++ coding style in the main CODING_STYLE.md?